test/remote/gateways/remote_priority_test.rb

```diff
@@ -13,13 +13,18 @@ def setup
 
     # Run specific remote test
     # ruby -Itest test/remote/gateways/remote_priority_test.rb -n test_fail_refund_already_refunded_purchase_response
+
+    # If you are attempting to implement `replayId` on `refund` be aware of this caveat:
+    # When attempting duplicate refunds with the same `replayId` gsf, Priority returns this error: "Replay payment account #number does not match original." This is because the payment token is not associated with card information that is `stored` at Priority. When a card is vaulted at Priority and a duplicate refund is attempted the error returned will be "Payment already refunded". Customers should be made aware that using the GSF with a stored vs unstored card will return different results, though the final outcome of both should be a failed duplicate refund.
```

Comment:
This feels like a long winded comment that people will probably gloss over and is only specific to the `replay_id` field. I don't think we necessarily need to explain _why_ we are doing things in test, but we could maybe link to the documentation for the specific test case for this field.

Reply:
I think a better way to handle this is to use tests to show how this behavior works, which it looks like you are already doing with your test on ln306 and ln318.

Reply:
@naashton I see your point. The docs aren't specific about `refund` and `void`. I only learned this through my conversation with their support. I had originally written tests for `refund` and `void` but since they won't be usable with this GSF I didn't know that it was necessary. The response when the GSF is added is more confusing than leaving it out. Do you think I should add the `refund`/`void` tests back in or remove the comment?

Reply:
I think it makes sense to add the failed test for refund, regardless of whether the response message is different for a `stored` vs `unstored` card. It'll still convey the same sentiment around the behavior.